src/container/ImportCAT/ImportCAT.tsx

```diff
@@ -36,6 +42,16 @@ function ImportCAT() {
         [keyword, availableAssets.data]
     )
 
+    const handleRemoveSelected = (id: string) => {
+        const newSelected = selected.filter((item) => item.asset_id !== id)
+
+        setSelected(newSelected)
+
+        if (selected.length === 1) {
```

Comment:
I think `newSelected===0` is more 
easy to understand than `selected.length === 1`